### PR TITLE
Improve QoL of Next.js getting started pages

### PR DESCRIPTION
**Pull request by lforst**
- Fix some links in Next.js pages that unnecessarily lead to the `javascript` platform docs
- Minor rewordings
- Fix TypeScript error in code snippet
- Move paragraph that was at wrong location

### CODE REVIEW

File: src/platform-includes/getting-started-verify/javascript.nextjs.mdx [hunk 1]
```diff
@@ -38,6 +38,6 @@ export default withSentry(handler);
 
 <Note>
 
-Errors triggered from within Browser DevTools are sandboxed, so will not trigger an error handler. Place the snippet directly in your code instead.
```
Comment (lforst) [inline src/platform-includes/getting-started-verify/javascript.nextjs.mdx:41]: You can't paste the snippets above in the console --> this sentence doesn't make sense.

Also, why does the user care about an error handler not being triggered? They most likely care about error monitoring not being triggered --> reworded

File: src/platform-includes/getting-started-primer/javascript.nextjs.mdx [hunk 2]
```diff
@@ -15,4 +15,4 @@ Features:
 - Automatic [Performance Monitoring](/product/performance/) for both the client and server, from version `6.5.0`
 
 Under the hood the SDK relies on our [React SDK](/platforms/javascript/guides/react/) on the frontend and [Node SDK](/platforms/node) on the backend, which makes all features available in those SDKs also available in this SDK.
-However, using a [custom server](https://nextjs.org/docs/advanced-features/custom-server) is not currently supported.
```
Comment (lforst) [inline src/platform-includes/getting-started-primer/javascript.nextjs.mdx:18]: "However" didn't make sense here

File: src/platforms/javascript/guides/nextjs/manual-setup.mdx [hunk 3]
```diff
@@ -32,8 +32,7 @@ Sentry.init({
 ```typescript {filename:sentry.server.config.ts/sentry.client.config.ts}
 import * as Sentry from "@sentry/nextjs";
 
-const SENTRY_DSN: string =
-  process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
```
Comment (lforst) [inline src/platforms/javascript/guides/nextjs/manual-setup.mdx:36]: This caused a TypeScript error

File: src/platforms/javascript/guides/nextjs/manual-setup.mdx [hunk 4]
```diff
@@ -70,8 +71,6 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
 export default withSentry(handler);
 ```
 
-You can include your DSN directly in these two files, or provide it in either of two environment variables, `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`.
-
```
Comment (lforst) [inline src/platforms/javascript/guides/nextjs/manual-setup.mdx:74]: This paragraph was misplaced.

**Review by AbhiPrasad — APPROVED** [review]

**Pull request merged**